vignettes/cmdstanr.Rmd

```diff
@@ -194,23 +194,35 @@ fit$summary("theta", pr_lt_half = ~ mean(. <= 0.5))
 ### Posterior draws
 
 The [`$draws()`](https://mc-stan.org/cmdstanr/reference/fit-method-draws.html)
-method can be used to extract the posterior draws as a 3-D array (iteration x
-chain x variable). The **posterior** package can then be used to easily convert
-to other formats, like data frames and matrices of draws.
+method can be used to extract the posterior draws in formats provided by the
+[**posterior**](https://mc-stan.org/posterior/) package. Here we demonstrate
+only the `draws_array` and `draws_df` formats, but the **posterior** package
+supports several other useful formats as well.
 
 ```{r draws, message=FALSE}
-# this is a draws_array object from the posterior package
-draws_array <- fit$draws()
-str(draws_array)
-
-# convert to matrix or data frame 
-draws_df <- as_draws_df(draws_array) # as_draws_matrix() for matrix
+# default is a 3-D draws_array object from the posterior package
+# iterations x chains x variables
+draws_arr <- fit$draws() # or format="array"
+str(draws_arr)
+
+# draws x variables data frame
+draws_df <- fit$draws(format = "df")
+str(draws_df)
 print(draws_df)
 ```
 
+To convert an existing draws object to a different format use the
+`posterior::as_draws_*()` functions.
+
+```{r as_draws}
+# this should be identical to draws_df created via draws(format = "df")
```

Comment:
Maybe mention that subsequent converts will be slower than just setting the appropriate format in the draws() call. It wont matter in most cases though.

Reply:
Good call, but I guess it kind of depends on which option we decide on for solving #523 right?

Reply:
Yeah, if we decide to convert on return for draws matrix it will be the same for draws matrix.

Reply:
> Maybe mention that subsequent converts will be slower than just setting the appropriate format in the draws() call. It wont matter in most cases though.

Coming back to this now finally. Just added this: 

> In general, converting to a different draws format in this way will be slower
than just setting the appropriate format initially in the call to the `$draws()`
method, but in most cases the speed difference will be minor.